### DISCUSSION TITLE
Registered for IETF 116 Hackathon

### DISCUSSION THREAD

**Discussion by johngray-dev** [IETF-Hackathon/pqc-certificates#29] [General]
I registered our team for the IETF 116 hackathon!  Our project is listed on the IETF 116 Hackathon wiki page now:

https://wiki.ietf.org/e/en/meeting/116/hackathon

I put myself and Mike and Champions, does anyone else want to be listed as a Champion?    You are all champions in my mind...  :)

**Comment by tomofumiokubo** [comment]
Hello John,

I have a potential work item around cert binding for the next Hackathon.
Please add me as a Champion!

Cheers,
Tomofumi
